### PR TITLE
Batch version delete

### PR DESCRIPTION
**Pull request by jaschrep-msft**
Delete specific blob versions via batch

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs [hunk 1]
```diff
@@ -19,6 +19,9 @@
 using NUnit.Framework;
 using System.Text.RegularExpressions;
 using Azure.Core.TestFramework.Models;
+using Azure.Storage.Tests.Shared;
+using Gee.External.Capstone.M68K;
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs:23]: seems accidentally added in

File: sdk/storage/Azure.Storage.Blobs/tests/ClientExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+using System.Threading;
+using System.Threading.Tasks;
+using Azure.Core.Pipeline;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+
+namespace Azure.Storage.Blobs.Tests;
+
+/// <summary>
+/// Data movement extension methods for the <see cref="Azure.Storage.Blobs.BlobContainerClient"/>.
```
Comment (amnguye) [inline sdk/storage/Azure.Storage.Blobs/tests/ClientExtensions.cs:15]: NIT - Remove Data movement

**Review by amnguye — APPROVED** [review]

**Review by seanmcc-msft — APPROVED** [review]
Can we update the changelog?

File: sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs (hunk 1 above)
Comment (jaschrep-msft) [reply sdk/storage/Azure.Storage.Blobs.Batch/tests/BlobBatchClientTests.cs:23]: thanks visual studio, what even is that second one?

**Pull request merged**